promtail/apparmor.txt

```diff
@@ -84,6 +84,7 @@ profile promtail flags=(attach_disconnected,mediate_deleted) {
     @{PROC}/sys/net/core/somaxconn        r,
     @{sys}/kernel/mm/transparent_hugepage/hpage_pmd_size r,
     /dev/null                             k,
+    /etc/ssl/certs/**    r,
```

Comment:
```suggestion
    @{do_etc}/ssl/certs/**    r,
```
Change it to this instead. I found in testing that docker behaves differently on some systems. Sometimes when a docker container tries to read the file it uses the path as the container sees it (starting from `/`) and sometimes lower level commands use the path as the host sees it (starting from `/docker/overlay2/...`). The variable normalizes it so it works either way.